### PR TITLE
Remove vestigial FlowControl code for clarity

### PR DESCRIPTION
**Pull request by benclifford**
This code is apparently unused.

### CODE REVIEW

File: parsl/dataflow/dflow.py [hunk 1]
```diff
@@ -182,11 +182,6 @@ def __init__(self, config=Config()):
                 logger.error("invalid checkpoint_period provided:{0} expected HH:MM:SS".format(config.checkpoint_period))
                 self._checkpoint_timer = Timer(self.checkpoint, interval=(30 * 60))
 
-        if any([x.managed for x in config.executors]):
```
Comment (yadudoc) [inline parsl/dataflow/dflow.py]: Hey Ben, I believe we start a flowcontrol thread if any of the executors passed in are managed by Parsl. The flowcontrol thread is what i think calls the strategy piece at some interval.

**Comment by benclifford** [comment]
Hm. I think you are right.

**Comment by yadudoc** [comment]
I don't think we can remove this without redoing code touching strategy/flowcontrol. So let's scrap this PR ?

**Comment by benclifford** [comment]
yep, i agree

**Pull request closed without merging**